src/assets/css/footer.scss

```
@@ -0,0 +1,206 @@
+body > p {
+  background-color: $moramia-blue;
+  height: 100vw;
+}
+* {
+  box-sizing: border-box;
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+  text-decoration: none;
+}
+.footer {
+  background-color: $footer-background;
+  bottom: 0;
+  color: $moramia-white;
+  display: block;
+  left: 0;
+  overflow: hidden;
+  padding: 5em 0;
+  position: relative;
+  transition: all 0.4s ease;
+  width: 100vw;
+}
+.footer-content {
+  display: block;
+  font-family: $font-stack;
+  height: 100%;
+  margin: auto auto;
+  padding: 3em;
+  .footer-cp {
+    color: $cp-color;
+    font-size: 0.8em;
+    margin: 1.5em 0;
+    padding: 0.7em 0;
+  }
+}
+.menu-wrap {
+  z-index: 1;
+  .toggler {
+    cursor: pointer;
+    height: 30px;
+    position: absolute;
+    opacity: 0;
+    width: 30px;
+    z-index: 2;
+  }
+  .hamburger {
+    align-items: center;
+    background-color: $moramia-blue;
+    display: flex;
+    height: 30px;
+    padding: 0.75rem;
+    position: absolute;
+    width: 35px;
+    z-index: 1;
+  }
+}
+.footer-image,
+.menu-wrap {
+  margin: auto;
+  max-width: 550px;
+}
+
+.footer-center-line {
+  align-items: center;
+  background-color: $moramia-white;
+  display: flex;
+  height: 1.7px;
+  justify-content: center;
+  position: relative;
+  transition: all 0.4s ease;
+  width: 100%;
+}
+/*top and bottom lines*/
+.menu-wrap .hamburger > div:before,
+.menu-wrap .hamburger > div:after {
+  background: $moramia-white;
+  content: "";
+  height: 1.7px;
+  position: absolute;
+  top: -4px;
+  width: 100%;
+  z-index: 1;
+}
+.menu-wrap .hamburger > div:after {
+  top: 4px;
+}
+/*toggler animation*/
+.menu-wrap .toggler:checked + .hamburger > div {
+  transform: rotate(135deg);
+}
+
+.menu-wrap .toggler:checked + .hamburger > div:before,
+.menu-wrap .toggler:checked + .hamburger > div:after {
+  top: 0;
+  transform: rotate(95deg);
+}
+
+/*Rotate on hover checked*/
+.menu-wrap .toggler:checked:hover + .hamburger > div {
+  transform: rotate(225deg);
+}
+.menu-wrap {
+  .menu {
+    max-height: 0;
+    opacity: 1;
+    overflow: hidden;
+    position: relative;
+    top: 40px;
+    transition: all 0.8s ease;
+    visibility: hidden;
+    width: 100%;
+  }
+}
+a {
+  color: $moramia-white;
+  transition: all 0.8s ease;
+}
+.footer-home {
+  color: $moramia-blue;
+  &:hover {
+    color: $moramia-white;
+  }
+}
+.footer-list {
+  > li {
+    > a {
+      &:hover {
+        color: $moramia-blue;
+        transition: all 0.4s ease;
+      }
+    }
+    margin: 2em 0;
+  }
+}
+.menu-wrap {
+  .toggler {
+    &:checked {
+      ~ {
+        .menu {
+          height: 100%;
+          max-height: 500px;
+          opacity: 1;
+          transition: all 1s ease;
+          visibility: visible;
+        }
+      }
+    }
+  }
+}
+
+@media screen and (min-width: $breakpoint-tablet) {
+  .footer {
+    height: 100%;
+    padding: 1em;
+    transition: all 0.4s ease;
+  }
+  .footer-content {
+    align-items: center !important;
```

```suggestion
    align-items: center;
```
You don't need !important, double check the cascade.

---

Done